quill-core/src/test/scala/io/getquill/quotation/QuotationSpec.scala

```diff
@@ -684,6 +684,13 @@ class QuotationSpec extends Spec {
         }
         quote(unquote(q)).ast.body mustEqual OptionOperation(OptionExists, Ident("o"), Ident("v"), Ident("v"))
       }
+      "contains" in {
```

Comment:
I do agree, I need to add more tests. But I would like your opinion on the solution first.